src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2911,7 +2913,7 @@ void NetworkOPsImp::getBookPage (
 // FIXME : support iLimit.
 void NetworkOPsImp::getBookPage (
     bool bAdmin,
-    Ledger::pointer lpLedger,
+    std::shared_ptr<ReadView const>& lpLedger,
```

Comment:
Sorry for the mistake - the code is not compiled so such errors get ignored.

We should really get rid of these, they've been here for over a year.